Integrations/SecBI/SecBI.py

```diff
@@ -0,0 +1,193 @@
+        :return: SecBI Incident IDs List
+        """
+        params = assign_params(query=query, limit=limit)
+        LOG('Performing SecBI get_incidents_list request to {}/{} with params={}'.format(self._base_url,
+                                                                                         SECBI_API_ENDPOINT_INCIDENTS,
+                                                                                         str(params)))
+        return self._http_request(method='GET',
+                                  url_suffix=SECBI_API_ENDPOINT_INCIDENTS,
+                                  params=params)
+
+    def secbi_get_incident(self, incident_id):
+        """
+        Get a specific SecBI Incident by SecBI Incident ID
+        :param incident_id: SecBI incident ID
+        :return: A dictionary representation of the SecBI Incident requested
+        """
+        params = assign_params(incidentId=incident_id, fields=SECBI_INCIDENT_FIELDS)
+        return self._http_request(method='GET',
+                                  url_suffix='{}/{}'.format(SECBI_API_ENDPOINT_INCIDENT, incident_id),
+                                  params=params)
+
+    def secbi_get_incident_by_host(self, host):
+        """
+        Get a specific SecBI Incident by Host
+        :param host: The host by which to get a SecBI Incident
+        :return:  A dictionary representation of the SecBI Incident found to contain the supplied host
+        """
+        params = assign_params(host=host, fields=SECBI_INCIDENT_FIELDS)
+        return self._http_request(method='GET', url_suffix=SECBI_API_ENDPOINT_INCIDENT, params=params)
+
+    def test_module(self):
+        """
+        Performs basic get request to test SecBI system
+        """
+        return self._http_request(method='GET', url_suffix=SECBI_API_ENDPOINT_STATUS)
+
+
+def test_module_command(client: SecBIClient):
+    """
+    This is the call made when pressing the integration test button.
+    Performs basic get request to test SecBI system
+    :param client: The SecBI client to use for the sanity test
+    """
+    results = client.test_module()
+    if 'status' in results and results['status'] == 'ACTIVE':
+        demisto.results('ok')
+    raise DemistoException("Failed connection test of SecBI. Please check your SECBI_API_URL and SECBI_API_KEY")
+
+
+def secbi_get_incidents_list_command(client: SecBIClient, args: Dict) -> Tuple[str, Dict, Dict]:
+    """
+    SecBI Get All Incident IDs command
+    :param client: The SecBI client to use
+    :param args: The Demisto args
+    :return: Content for return_outputs()
+    """
+    query = args.get('query', None)
+    limit = args.get('limit', 100)
+    incidents_list = client.secbi_get_incidents_list(query, limit)
+
+    formatted = [dict(ID=s) for s in incidents_list]
+    human_readable = tableToMarkdown('List of SecBI Incidents', formatted)
+    entry_context = {
+        'SecBI.IncidentsList': incidents_list
```

Comment:
```suggestion
        'SecBI.Incident': incidents_list
```

Reply:
This isn't the same thing. The `IncidentsList` only contains a list of IDs (strings). Where as the `Incident` contains a JSON of an actual Incident's data.

Reply:
arent these the same objects? like same incidents? the ids are the same ?

Reply:
Each ID in the `IncidentsList` corresponds to an actual SecBI Incident that can be requested via the `secbi-get-incident` command and then the actual Incident JSON will be retrieved and saved to the `Incident` context.

Reply:
then both commands should populate outputs under the same context path: `SecBI.Incident`